proposals/kubeflow-technical-oversight-committee.md

```diff
@@ -0,0 +1,184 @@
+# Kubeflow Technical Oversight Committee
+
+The Kubeflow Technical Oversight Committee (TODO: TOC or KTOC ?) is responsible for
+cross-cutting product and design decisions.
+
+The governance of Kubeflow is an open, living document, and will continue to evolve as the
+community and project change.
+
+## Charter
+
+- Project direction and roadmap:
+
+  - Set the overall technical direction and Kubeflow roadmap.
+
+  - Create proposals based on TOC discussions and bring them to the relevant working groups for discussion.
+
+  - Working with working groups on Kubeflow cross-component integrations.
+
+  - Resolving issues that, despite best efforts, a working group has been unable to resolve.
+
+- Project releases, security, and stability:
+
+  - Ensure the stability of Kubeflow releases.
+
+  - Set the priorities of individual releases to ensure coherency and proper sequencing.
+
```

Comment:
I believe the Release Manager has been the final authority for a release.   We can change but perhaps the final authority / decision maker needs to be defined.   I believe we can have many inputs and influencers, but only 1 decision maker.

Reply:
+1

Reply:
+1 - Agree. The TOC will definitely influence what WGs decide to implement and propose in their roadmap, but ultimate accountability for roadmap readiness should be on the release team

Reply:
@jbottum @StefanoFioravanzo @johnugeorge Any thoughts on who should be the final authority for Kubeflow releases ?

Reply:
Depends on whether we want to make the TOC just a "consulted" party or ultimately accountable for the roadmap. Being the primary technical body, I am inclined to say that the TOC should have final approval of the roadmap proposed by the release manager/team. This could become a new action item in the release handbook. 

Another way to look at this is: pushing accountability to the Release Manager could be done by having the TOC appoint the Release Manager in the first place, delegating that responsibility.

Reply:
Maybe let's look at how other communities are doing this